dll.webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const webpack = require('webpack')
+const path = require('path')
+
+module.exports = {
+  entry: {
+    vendor: [
+      'prop-types',
```

Comment:
Someone should make something that just lets you say vendor: path.join(__dirname, 'node_modules')

Reply:
Yeah, there are some third party plugins that will do that (and a few other nice extras). I think it's a little tricky because there might be some node modules that it's a bad idea to include in the bundle (though I can't really think of any good examples off the top of my head).